www/js/pedagogical-module.js

```diff
@@ -132,6 +133,12 @@ define([
 
 	function message(/*object*/ obj, /*string*/ nodePart, /*string*/ status){
 		// TO DO : Add Hint messages
+		record.increment(status, 1);
+		if(status === "extra" || status === "irrelevant"){
+			status = "incorrect";
+		}else if(status === "lastFailure" || status === "lastFailure2"){
+			status = "incorrect. The correct answer has been given";
+		}
```

Comment:
I have already moved this message out to a file. Is there a reason for it to be added again?

Reply:
Removed these